.github/workflows/build-and-deploy.yml

```diff
@@ -364,14 +370,16 @@ jobs:
               yum -y localinstall "$installer"
           done
       - name: Test installation
-        run: /opt/alchemist/bin/alchemist --help
+        run: alchemist --help
 
   # Test PKGBUILD for AUR release
   verify-aur-package:
     needs:
       - ci-preparation
       - assemble-and-upload
     runs-on:  ubuntu-22.04
+    if: >-
+      always() && contains(needs.*.result, 'success')
```

Comment:
why?

Reply:
To prevent tests to be skipped when staging-repo is not executed, see https://github.com/Zimbrando/Alchemist/actions/runs/7817743725 referring to this commit https://github.com/AlchemistSimulator/Alchemist/pull/3120/commits/1e571ccb265c84dc383573d43106dc6f50f12b0c